docs/docs-content/clusters/edge/trusted-boot/trusted-boot.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: "Trusted Boot"
+title: "Trusted Boot"
+description: "Learn about Trusted Boot."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["edge"]
+---
+
+Trusted Boot is a security feature supported by Palette Edge running on Edge devices with supported hardware and
+firmware. Trusted Boot consists of the following security measures:
+
+- Full disk encryption (FDE): Encryption of the entire contents of a disk drive, including the operating system, system
```

Comment:
unfortunately this is not technically correct. we dont encrypt the entire disk.  in particular we dont encrypt the  operating system. we encrypt all sensitive user data (persistent partition) securely, with a minimalistic boot image (UKI) that is immutable and cannot be tampered with . The TPM releases the key for decrypting the persistent data based off the measurements kept during boot.

Reply:
If it's not full disk encryption, maybe we shouldn't call it full disk encryption? Maybe persistent disk encryption? I feel like calling it full disk encryption would make it confusing to users

Reply:
can you take a look at perhaps explaining that FDE is an industry term?